### DISCUSSION TITLE
Possible to force dark theme? (Cockpit ignores both system and browser being set to dark and remains light)

### DISCUSSION THREAD

**Discussion by hugalafutro** [cockpit-project/cockpit#18441] [Q&A] [answered]
Hi, I was looking for a switch to dark theme and didn't find one, but I found a blog post which said Cockpit will automatically switch to dark mode if system is in dark mode.

This is not the case for me. I have Win 11 in dark mode and use Chromium based browser (which itself is set to dark mode), but Cockpit interface is light. BUT a non-official Application I manually installed (Benchmarks) is in dark mode when I access it, however any other section will revert to light mode.

Am I just blind and is there a manual switch or is there a way to force dark mode?

Thanks for any info

**Comment by KKoukiou** [comment, marked as answer]
Hi, the dark theme switcher was added in cockpit 281. See https://cockpit-project.org/blog/cockpit-281.html

**Reply by hugalafutro** [reply]
I see, I incorrectly assumed the ubuntu repo holds the latest version of Cockpit, my bad. It installed v264. Thanks for lightning quick response!

**Reply by KKoukiou** [reply]
You can get a newer release from backports if you want https://cockpit-project.org/running.html#ubuntu
